src/main/java/io/confluent/connect/jdbc/TimestampIncrementingTableQuerier.java

```diff
@@ -147,18 +149,25 @@ protected void createPreparedStatement(Connection db) throws SQLException {
   @Override
   protected ResultSet executeQuery() throws SQLException {
     if (incrementingColumn != null && timestampColumn != null) {
-      Timestamp ts = new Timestamp(timestampOffset == null ? 0 : timestampOffset);
-      stmt.setTimestamp(1, ts, UTC_CALENDAR);
-      stmt.setLong(2, (incrementingOffset == null ? -1 : incrementingOffset));
-      stmt.setTimestamp(3, ts, UTC_CALENDAR);
-      log.debug("Executing prepared statement with timestamp value = " + timestampOffset + " and incrementing value = " + incrementingOffset);
+      Timestamp startTime = new Timestamp(timestampOffset == null ? 0 : timestampOffset);
+      Timestamp endTime = new Timestamp(new Date().getTime() - timestampDelay);
```

Comment:
Hmm, so I thought about this when doing the initial implementation. I avoided using any timestamps from the node running this since then we're susceptible to time sync issues between the database and the host running the connector. And that is unfortunately pretty painful to unit test for correctness. Is it possible to rewrite this to do that computation via the SQL statement?